transaction.js

```diff
@@ -38,6 +38,7 @@ class Transaction {
             tempfail: 0,
             reject: 0,
         }
+        this.msg_status = undefined;
```

Comment:
Why not `null` like other fields?

Reply:
Ask the other person of this discussion. But basically, because it's a good English word, it completely reflects the status.

Reply:
FTR, I didn't choose `undefined` over `null`, I suggested that `undefined` the special JS value was better than `'undefined'` as a string. If `null` is more consistent with the surrounding code, and there's no compelling reason to prefer `undefined`, then null is preferable. In this case, I think it doesn't matter to Kibana/ES because the ES plugin prunes both `null` and `undefined`.